src/openrct2/interface/Window.h

```diff
@@ -43,14 +43,14 @@ enum class CloseWindowModifier : uint8_t;
 constexpr uint8_t CloseButtonWidth = 10;
 
 #define SCROLLABLE_ROW_HEIGHT 12
```

Comment:
I left this untouched as it is being changed in this PR https://github.com/OpenRCT2/OpenRCT2/pull/21760